nss-day-cohort-51/nutshell-ghoul-diggers
# Feature/nutshell ss 10

# Description:

### I created an EventsList component that displays a page of events from the database. Each event is represented by a card. I also created a form for user to input new events and save it to their database. EventsList page filters them by date and displays them in Upcoming and Past Events. Each card is able to be Edited and Deleted. Each card also displays a Weather button that when pressed pops up a modal with the forecast for the zipcode and date of the event as long as it's within the next 10 days. 

## Type of change


# Testing Instructions
Serve page with npm start, click on Events in the NavBar. Click on buttons next to card to edit or delete post, then click on the weather button to make sure it pops up and displays forecast.

# Checklist:
